### ISSUE TITLE
Error due to empty response from eris-db upon event subscription

### ISSUE THREAD

**Issue by j-h-scheufen** [monax/legacy-contracts.js#73]
# Version Numbers

* Node.js: 7.4.0
* Eris CLI: 0.12.0
* eris-contracts: 0.15.9 

# Issue Description

Subscribing to two events from a node application causes the following errors (or warnings?) to be logged.
```
Call of Eris DB method 'eventSubscribe' with parameters {
  "event_id": "Log/313AAAFDFE26591B502C6A2E16CAABB740F38BDA"
} responded with "".
Call of Eris DB method 'eventSubscribe' with parameters {
  "event_id": "Log/313AAAFDFE26591B502C6A2E16CAABB740F38BDA"
} responded with "".
(node:77363) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error
(node:77363) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:77363) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error
Call of Eris DB method 'eventSubscribe' with parameters {
  "event_id": "Log/313AAAFDFE26591B502C6A2E16CAABB740F38BDA"
} responded with "".
Call of Eris DB method 'eventSubscribe' with parameters {
  "event_id": "Log/313AAAFDFE26591B502C6A2E16CAABB740F38BDA"
} responded with "".
(node:77363) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 3): Error
(node:77363) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 4): Error
```

# Code

The code subscribing to the events follows the documentation and can be seen in the following snapshot. `registry` is an instantiated contract object and `EntityCreated` is the event.
```
    self.registry.EntityCreated(function(error, eventSub) {
        if(error) {
            throw error;
        }
        subscriptionCreateEvent = eventSub;
    },
    function(err, event) {
        if(err || !event) return;

        self.eventSubscriptions["EntityCreated"].forEach(function(subscriber) {
            subscriber(err, processEvent(event))
        });
    });

    // for each 'EntityUpdated' subscriber, convert event to a JSON object and reply
    self.registry.EntityUpdated(function(error, eventSub) {
        if(error) {
            throw error;
        }
        subscriptionUpdateEvent = eventSub;
    },
    function(err, event) {
        if(err || !event) return;

        self.eventSubscriptions["EntityUpdated"].forEach(function(subscriber) {
            subscriber(err, processEvent(event));
        });
    });
```

This error appears in two separate projects. It is reproducible by running the apps in these repositories:
https://github.com/eris-ltd/app-lei-ledger/tree/feature/stage-1
https://github.com/eris-ltd/hello-eris/tree/feature-10-timeout-fix/apps/hello-eris

**Comment by NodeGuy** [comment]
I can't reproduce the contract deployment and created an issue for CLI (https://github.com/eris-ltd/eris/issues/1192).

Please reproduce the bug in Vagrant (https://github.com/eris-ltd/eris-vagrant).

**Comment by NodeGuy** [comment]
There was a bug blocking the display of the error message so you could see only `responded with ""`.  That bug has been fixed in 0.15.12; please upgrade and try again.
